packages/library-base/src/commonMain/kotlin/io/realm/kotlin/log/RealmLogger.kt

```diff
@@ -38,12 +39,23 @@ public interface RealmLogger {
     /**
      * Log an event.
      */
-    public fun log(level: LogLevel, throwable: Throwable?, message: String?, vararg args: Any?)
+    public fun log(category: LogCategory, level: LogLevel, throwable: Throwable?, message: String?, vararg args: Any?) {
+        log(level, throwable, message, *args)
```

Comment:
Should we consider padding the category on the messages in the default implementation with: `message = "[$category]$message"`?

Reply:
Great point, the categories could even be used as tags.

Reply:
It might be that there is some issues with mixing it up with the tags. On Android there are other constraints on the tags, etc., so seems safest to just pad it with `[$category] $message`

Reply:
Yes, there is a limit on tag length... :(